Formula/gomplate.rb

```diff
@@ -0,0 +1,33 @@
+class Gomplate < Formula
+  desc "Command-line Golang template processor"
+  homepage "https://gomplate.hairyhenderson.ca/"
+  url "https://github.com/hairyhenderson/gomplate/archive/v1.9.1.tar.gz"
+  sha256 "4d642bcb6d72b793dadaa622b4a2f42efea1f2425f428798f1feacdbf158f812"
+  head "https://github.com/hairyhenderson/gomplate.git"
+
+  depends_on "glide" => :build
+  depends_on "go" => :build
+  depends_on "upx" => :build
```

Comment:
It can be removed but it will increase the size of the cask.

UPX output:

```
                       Ultimate Packer for eXecutables
                          Copyright (C) 1996 - 2017
UPX 3.94        Markus Oberhumer, Laszlo Molnar & John Reiser   May 12th 2017

        File size         Ratio      Format      Name
   --------------------   ------   -----------   -----------
  16043316 ->   4223632   26.33%   macho/amd64   gomplate-slim

Packed 1 file.
```

So 16mb is compressed to 4mb.

Your call: I can easily change it.